File: src/gauntlet/dto/PlayerControllerSettingsDto.ts
//Espelho de PlayerControllerSettingsDto.java (GET /api/player-controller-settings/{character}).
export interface PlayerControllerSettingsDto {
    moveSpeedForward: number;
    moveSpeedBackward: number;
    accel: number;
    angularVelocityDegPerSec: number;
    moveStateEpsilon: number;
    playerRadius: number;
    idleTurnMultiplier: number;
}

File: gauntlet_server/gauntlet/src/main/java/net/dongeronimo/gauntlet/interfaces/transferObjects/PlayerControllerSettingsDto.java
package net.dongeronimo.gauntlet.interfaces.transferObjects;

/** Resposta de GET /api/player-controller-settings/{character} — o client usa isto pra
 *  calibrar a predição local (MineAvatarBehaviour) com os MESMOS valores que
 *  o server está rodando (ver GameLoop, PlayerControllerSettings). */
public record PlayerControllerSettingsDto(
    double moveSpeedForward,
    double moveSpeedBackward,
    double accel,
    double angularVelocityDegPerSec,
    double moveStateEpsilon,
    double playerRadius,
    double idleTurnMultiplier
) {}
